user.php

```diff
@@ -0,0 +1,61 @@
+<?php
+	include("connection.php");
+	/**
+	* @author: mfsi_krishnadev
+	* @class: validating user
+	*/
+	class user extends db_connection
+	{
+		
+		/*
+		 * @access: public
+		 * @param : email-id, password 
+		 * @return type: none
+		*/
+
+		function check($mail_id, $pass)
+		{
+
+			$email_id = $this->test_input($mail_id);
+			$password = md5($pass);
+
+
+			$select = "SELECT id, CONCAT(first_name, ' ',last_name)as name FROM employee
+				WHERE email_id = '$email_id' AND password = '$password'";
+
+			$query = mysqli_query($this->connect, $select);
+			$row = mysqli_num_rows($query);
+
+			if($row == 1)
```

Comment:
- Always use variables on the right side and strings/constant of the left side.
if($row == 1) should be written as if(1 === $row)
- Always use type checking that is === instead of ==, !== instead of !=